internal/dataplane/sendconfig/sendconfig.go

```diff
@@ -37,26 +34,6 @@ type AdminAPIClient interface {
 	KonnectControlPlane() string
 }
 
-// Note that UpdateError is used for both DB-less and DB-backed update strategies, but does not have parity between the
-// two. Pending a refactor of database reconciler (https://github.com/Kong/go-database-reconciler/issues/22), KIC does
-// not have access to per-resource errors or any original response bodies in database mode. Future DB mode work would
-// need to find some way to reconcile the single /config raw body and many per-resource responses from DB endpoints.
```

Comment:
I removed it because I concluded that https://github.com/Kong/go-database-reconciler/pull/76 effectively resolved this comment - we're now getting per-resource feedback from GDR via a feedback channel. cc @rainest